Assets/MRTK/Providers/WindowsMixedReality/XRSDK/Controllers/WindowsMixedRealityXRSDKMotionController.cs

```diff
@@ -26,7 +26,7 @@ public WindowsMixedRealityXRSDKMotionController(TrackingState trackingState, Han
         {
             new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
             new MixedRealityInteractionMapping(1, "Spatial Grip", AxisType.SixDof, DeviceInputType.SpatialGrip),
-            new MixedRealityInteractionMapping(2, "Grip Press", AxisType.SingleAxis, DeviceInputType.TriggerPress),
+            new MixedRealityInteractionMapping(2, "Grip Press", AxisType.Digital, DeviceInputType.GripPress),
```

Comment:
Does this break the controller mapping sync? or do we already ignore these values when trying to align actions?

Reply:
this should be singleaxis since we still wanted the float data

Reply:
I was thinking about this code, but looks like we just assume the exact same ordering:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/31f1a04118d3a35a1c7855d0a01034bfa7a0cb77/Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerMapping.cs#L239-L249